components/automate-gateway/handler/iam/v2beta/policy/policy.go

```diff
@@ -406,7 +412,7 @@ func (p *Server) GetProject(
 	return &pb_resp.GetProjectResp{Project: proj}, nil
 }
 
-// ListProjects returns the list of all projects.
+// ListProjects returns the list of all projects for display on the List Projects view.
```

Comment:
I'm curious about this comment-- `ListProjects` isn't used elsewhere? It's exposed, right?